### ISSUE TITLE
赋予APP网络权限后，还是一直提示”似乎已断开与互联网的连接”

### ISSUE THREAD

**Issue by mynamecool** [bytedance/Fastbot_iOS#109]
已经给应用设置网络权限了，但是还是一直提示“似乎已断开与互联网的连接”，不过倒是不影响使用。
![image](https://user-images.githubusercontent.com/40492728/194816435-abb7410b-ae9f-47f7-a6e7-e12b13c46926.png)

**Comment by qingtianwaa** [comment]
你好，项请问下你这个在 ping network时，有什么其他报错吗，是否可以交流下

**Comment by mynamecool** [comment]
是的 是 ping network 那个testcase 报的错，没有其他的报错，我尝试过修改字节官网的链接，一开始以为是重定向的问题，就将他的链接修改为了https，但是返回的字节数还是0，不过不影响正常执行monkey，主要是不知道他赋予网络权限的目的是什么，如果是没有什么影响，打包的时候 把这个case去掉也行

**Comment by caimengyuan** [comment]
我跟你一样，不过 ping network 这个步骤不影响测试
